tests/rendering/test_all_renderers_markdown_contract.py

```
@@ -0,0 +1,181 @@
+        if isinstance(obj, list):
+            return [poison(i) for i in obj]
+        if isinstance(obj, dict):
+            return {k: (v if k == "url" else poison(v)) for k, v in obj.items()}
+        return obj
+
+    offenders = []
+    for crew in RENDERABLE:
+        try:
+            html = RendererFactory.create_renderer(crew).render(poison(_payload()))
+        except Exception:
+            continue
```

Ruff (0.15.20)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)